samples/api-client/mqtt_example/README.rst

```diff
@@ -0,0 +1,151 @@
+.. This file is automatically generated. Do not edit this file directly.
+
+Google Cloud IoT Core Python Samples
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=iot/api-client/mqtt_example/README.rst
```

Comment:
It's pointing to `GoogleCloudPlatform/python-docs-samples` repo but the sample lives right here. I'm hoping this doesn't hardcode to `python-docs-samples` repo.

Reply:
Actually when I [fix](https://github.com/googleapis/python-iot/pull/194/commits/155e70f19b5b7689453be794187111dc0d311ebd) the repo path in the readme template, OwlBot automatically [changes it back](https://github.com/googleapis/python-iot/pull/194/commits/8fe6fbe11974e18aea3c9de3e0baddb6285c9de5) to python-docs-samples. I could not find where the value actually comes from.